Deploy Kafka on the Microsoft Azure Cobalt 100 processors (#16)
odidev/arm-learning-paths: content/learning-paths/servers-and-cloud-computing/kafka-azure/deploy.md

---
title: Install Kafka on Azure Cobalt 100 processors
weight: 4

### FIXED, DO NOT MODIFY
layout: learningpathall
---

## Install Kafka on Azure Cobalt 100

> This guide walks you through installing Apache Kafka 3.7.1 on an Azure Ubuntu Arm-based virtual machine. You’ll download Kafka, extract it into `/opt`, configure permissions, and verify the installation by checking the installed version.

Reviewer: ```suggestion
This guide walks you through installing Apache Kafka 3.7.1 on an Ubuntu Arm virtual machine. You’ll download Kafka, extract it into `/opt`, configure permissions, and verify the installation by checking the installed version.
```

---
title: Install Kafka
weight: 4

### FIXED, DO NOT MODIFY
layout: learningpathall
---

## Install Kafka on Azure Cobalt 100

This section walks you through installing latest version of Apache Kafka on an Ubuntu Arm virtual machine. You’ll download Kafka, extract it into `/opt`, configure permissions, and verify the installation by checking the installed version.

Follow the below instructions to install Kafka on Ubuntu virtual machine.

### Install Java

> Kafka requires Java to run. Install it using the following commands:

Reviewer 2: ```suggestion
Kafka requires Java to run. Install it by executing the following commands:
```

```console
sudo apt update
sudo apt install -y default-jdk
```
### Download and Install Kafka

> This sequence of commands downloads latest version(4.1.0) of Kafka to the `/opt` directory, extracts the tarball, renames the folder to kafka for simplicity, and sets ownership so the current user can access and manage the Kafka installation. It prepares the system for running Kafka without permission issues.

Reviewer: ```suggestion
This sequence of commands downloads Kafka version 4.1.0 to the `/opt` directory, extracts the tarball, renames the folder to kafka for simplicity, and sets ownership so the current user can access and manage the Kafka installation. It prepares the system for running Kafka without permission issues.
```
